services/todo-legacy/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "Example Todo list application",
+  "main": "index",
+  "scripts": {
+    "start": "ts-node index.ts",
+    "test": "mocha --compilers ts:ts-node/register,tsx:ts-node/register test/**/*.test.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "devDependencies": {
+    "@loopback/testlab": "^4.0.0-alpha.2",
+    "@types/mocha": "^2.2.41",
+    "mocha": "^3.3.0",
+    "ts-node": "^3.0.4",
+    "typescript": "^2.4.1"
```

Comment:
Sorry for the confusion in my previous comment. I meant that both `ts-node` and `typescript` should be regular dependencies, because they are needed to run the app in production (via `npm start`).